lib/test/python/test_nc_checker_call.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""
+unittest for lib/python/util.py netcdf checker call function
+
+author : besnard, laurent
+"""
+
+from util import pass_netcdf_checker
+import os
+import unittest
+
+class TestPassChecker(unittest.TestCase):
+
+    def setUp(self):
+        self.nc_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'test.nc')
+
+    def test_pass_netcdf_checker(self):
+        self.assertTrue(pass_netcdf_checker(self.nc_path, ['cf:1.6']))
+        self.assertTrue(pass_netcdf_checker(self.nc_path, ['imos:1.3']))
```

Comment:
@mhidas Are we going to implement proper versionning of the IMOS checker such as the ACDD one ?